### PR TITLE
[doc] add chinese version of README.md

### PR DESCRIPTION
**Pull request by dujuncheng**
this Chinese Version of README will helpful to people from china to understand and use this package

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
 # vuex-router-sync [![CircleCI](https://circleci.com/gh/vuejs/vuex-router-sync.svg?style=svg)](https://circleci.com/gh/vuejs/vuex-router-sync)
 
+
+[中文版本(Chinese Version)](/vuejs/vuex-router-sync/blob/master/README.zh-cn.md)
```
Comment (Jinjiang) [inline README.md]:
```suggestion
[中文版本 (Chinese Version)](/vuejs/vuex-router-sync/blob/master/README.zh-cn.md)
```

File: README.zh-cn.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
+## vuex-router-sync
+
+> 把 vue-router 当前的状态同步为 vuex 状态的一部分
```
Comment (Jinjiang) [inline README.zh-cn.md]:
```suggestion
> 把 vue-router 当前的 `$route` 同步为 vuex 状态的一部分。
```

File: README.zh-cn.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+### 用法
+
+```
+# 最新版本需要配合 vue-router 2.0及以上的版本使用
```
Comment (Jinjiang) [inline README.zh-cn.md]:
```suggestion
# 最新版本需要配合 vue-router 2.0 及以上的版本使用
```

File: README.zh-cn.md [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+npm install vuex-router-sync
+# 如果 vue-router 版本低于 2.0，执行如下
+npm install vuex-router-sync@2
+```
+
+
```
Comment (Jinjiang) [inline README.zh-cn.md]: don't need too many line-breaks here.

File: README.zh-cn.md [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+```javascript
+import { sync } from 'vuex-router-sync'
+import store from './vuex/store' // vuex store 实例
+import router from './router' // vue-router 实例
+
+const unsync = sync(store, router) // 返回值是 unsync 回调方法
+
+// 在这里写你的代码
+
+// 在 vue 应用销毁的阶段执行如下代码 (比如说, 你只在应用中一部分场景使用了 vue, 当跳出该场景想销毁 vue 实例时）
```
Comment (Jinjiang) [inline README.zh-cn.md]:
```suggestion
// 在 Vue 应用销毁时 (比如在仅部分场景使用 Vue 的应用中跳出该场景且希望销毁 Vue 的组件/资源时）
```

File: README.zh-cn.md [hunk 6]
```diff
@@ -0,0 +1,49 @@
+## vuex-router-sync
+
+> 把 vue-router 当前的状态同步为 vuex 状态的一部分
+
+### 用法
+
+```
+# 最新版本需要配合 vue-router 2.0及以上的版本使用
+npm install vuex-router-sync
+# 如果 vue-router 版本低于 2.0，执行如下
```
Comment (Jinjiang) [inline README.zh-cn.md]:
```suggestion
# 用于版本低于 2.0 的 vue-router
```

File: README.zh-cn.md [hunk 7]
```diff
@@ -0,0 +1,49 @@
...
+
+const unsync = sync(store, router) // 返回值是 unsync 回调方法
+
+// 在这里写你的代码
+
+// 在 vue 应用销毁的阶段执行如下代码 (比如说, 你只在应用中一部分场景使用了 vue, 当跳出该场景想销毁 vue 实例时）
+unsync() // 取消 store 和 router 中间的同步
+```
+
+你也可以设定一个自定义的 `vuex` `module`  的 `name` 值，如下：
```
Comment (Jinjiang) [inline README.zh-cn.md]:
```suggestion
你可以有选择地设定一个自定义的 vuex 模块名：
```

File: README.zh-cn.md [hunk 8]
```diff
@@ -0,0 +1,49 @@
...
+
+```javascript
+sync(store, router, { moduleName: 'RouteModule' } )
+```
+
+
```
Comment (Jinjiang) [inline README.zh-cn.md]: don't need too many line-breaks here.

File: README.zh-cn.md [hunk 9]
```diff
@@ -0,0 +1,49 @@
...
+
+### 工作原理
+
+- 该库在 `vuex`   `store` 上增加了默认名为 `route` 的 `module`, 用于表示当前路由的状态。
```
Comment (Jinjiang) [inline README.zh-cn.md]:
```suggestion
- 该库在 store 上增加了一个名为 `route` 的模块，用于表示当前路由的状态。
```

File: README.zh-cn.md [hunk 10]
```diff
@@ -0,0 +1,49 @@
...
+
+  ```javascript
+  store.state.route.path   // current path (字符串类型)
+  store.state.route.params // current params (对象类型)
+  store.state.route.query  // current query (对象类型)
+  ```
+
+- 当被导航到一个路由时，`store` 的状态会被同步的更新。
```
Comment (Jinjiang) [inline README.zh-cn.md]:
```suggestion
- 当被导航到一个新路由时，store 的状态会被更新。
```

File: README.zh-cn.md [hunk 11]
```diff
@@ -0,0 +1,49 @@
...
+
+- 请注意，`store.state.route` 是不可修改的，因为该值来源自 `URL` , 而不应该通过修改该值去触发浏览器的导航行为。相反，你只需要调用 `$router.push()` 或者 `$router.go()` 。另外，你可以通过 `$router.push({ query: {...}})` 来更新当前路径的 `query` 值。
```
Comment (Jinjiang) [inline README.zh-cn.md]:
```suggestion
- **`store.state.route` 是不可变更的，因为该值取自 URL，是真实的来源**。你不应该通过修改该值去触发浏览器的导航行为。取而代之的是调用 `$router.push()` 或者 `$router.go()`。另外，你可以通过 `$router.push({ query: {...}})` 来更新当前路径的查询字符串。
```

**Comment by kiaking** [comment]
@dujuncheng Do you think you have time to look into this?

**Comment by Jinjiang** [comment]
@kiaking seems the author not active. I just updated my all suggestions and approved it. Feel free to merge. Thanks.

**Review by Jinjiang — APPROVED** [review]

**Comment by kiaking** [comment]
Thanks a lot! 👍

**Pull request merged**